stories/slider/slider.stories.js

```diff
@@ -0,0 +1,146 @@
+export default {
+    title: 'Components/Slider',
+    parameters: {
+        description: `A slider is a control that enables the user to adjust single values within a specified numerical range. A range slider is a user interface control that enables the user to select a value range within a predefined numerical
+interval.`,
+        components: ['slider']
+    },
+    tags: ['f3', 'theme']
+};
+
+/**
+ * Basic slider
+ */
+
+export const basic = () => `
+<div class="slider-container">
+    <div class="fd-slider">
+        <div class="fd-slider__inner">
+            <div class="fd-slider__track">
+                <div class="fd-slider__track-range" style="width: 50%;"></div>
+                <div class="fd-slider__handle" tabindex="0" role="slider" aria-label="slider" aria-valuemin="1" aria-valuemax="100" aria-valuenow="50" style="left: 50%;"></div>
```

Comment:
After looking into this some more, this should really be an `input`. Using `div`s will put a large burden on the implementation libraries to handle keyboard and screen reader support. Mobile screen readers in particular seem to struggle with this. Using an `input` would make the experience consistent across all platforms and assistive technologies.

See the following examples:
https://butterpep.com/slider-accessibility.html
https://thepaciellogroup.github.io/AT-browser-tests/test-files/input-range.html
https://www.w3.org/TR/wai-aria-1.1/#slider

Reply:
> After looking into this some more, this should really be an `input`. Using `div`s will put a large burden on the implementation libraries to handle keyboard and screen reader support. Mobile screen readers in particular seem to struggle with this. Using an `input` would make the experience consistent across all platforms and assistive technologies.
> 
> See the following examples:
> https://butterpep.com/slider-accessibility.html
> https://thepaciellogroup.github.io/AT-browser-tests/test-files/input-range.html
> https://www.w3.org/TR/wai-aria-1.1/#slider

I got your point. However, I think that using `div` is OK in this case for a variety of reasons:
- poor styling abilities for `input[type="range"]` and inability to style in according to Fiori spec without both CSS/JS hacks
- implementing a range slider on native `input[type="range"]` will be way more complicated compared to `div`s
- handling keyboard/screen reader support and adding required `ARIA` attributes doesn't seem to be a large burden to me because frameworks provide in-built support of such functionality (Angular/React/Vue)
- existing implementation of an old Fiori slider is using `div`s (https://sapui5.hana.ondemand.com/1.82.4/#/entity/sap.m.RangeSlider/sample/sap.m.sample.RangeSlider)
- in addition, I couldn't find a popular 3rd party slider lib for Angular/React/Vue that uses `input[type="range"]`, IMO this is because of the first three reasons

> - https://material.angular.io/components/slider/overview
> - https://angular-slider.github.io/ng5-slider/
> - https://www.telerik.com/kendo-angular-ui/components/inputs/slider/
> - https://www.primefaces.org/primeng/showcase/#/slider
> - https://material-ui.com/components/slider/
> - https://zillow.github.io/react-slider/

Reply:
I'm not sure that other 3rd party libraries not doing this is necessarily a good reason. We had accessibility issues with a popular React datepicker that lead to us making improvements to the one here, porting it to fundamental-react, and using that instead, for example. Accessibility isn't always their number one concern. There are several UI5 components with accessibility issues as well.

And yes, React has tools that can help add better keyboard functionality but it is still a manual process to implement it; it doesn't come for free. It's unfortunate that the spec didn't take into consideration the native html markup that is built to support this and this design can't easily be implemented with the correct markup. 

If it's not possible to use inputs, the second best alternative is to ensure the documentation explicitly states the expected behavior for voice readouts and keyboard handling.

Reply:
Implementation libraries would still have to implement keyboard functionality regardless of what's used,`div` or `input` because spec describes such functionality as `jump` (incrementing/decrementing slider's value by a bigger step when `shift + arrow key` is pressed) which is something that native `input` doesn't provide.

Slider is an easier component to implement than a datepicker and I don't think adding a11y functionality will be a big problem (because of the fact that a lot of other libraries is using this approach and their implementation of a11y is pretty much the same which makes me think a11y isn't that hard to implement as it may seems)

Also I found W3 ARIA examples for sliders (both single and range) and they're using `div` with no mention that `input[type="range"]` is more preferable. 
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-1.html
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/multithumb-slider.html

I added a section in the docs that describes what attributes need to be added in order to make the slider accessible.

![image](https://user-images.githubusercontent.com/18414330/96869368-0821dc80-1478-11eb-9121-69fb4799d710.png)